composer.json

```diff
@@ -18,13 +18,13 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.3",
-        "symfony/framework-bundle": "~2.3",
-        "symfony/doctrine-bridge": "~2.3",
+        "symfony/framework-bundle": "~2.3|~3.0",
+        "symfony/doctrine-bridge": "~2.3|~3.0",
         "phpcr/phpcr-implementation": "2.1.*",
         "phpcr/phpcr-utils": "~1.2.0"
     },
     "require-dev": {
-        "symfony/form": "~2.2",
+        "symfony/form": "~2.3|~3.0",
```

Comment:
reverted. But you mean this bundle is using deprecated things, so this needs to be updated for best forward compat?